templates/docs/examples/patterns/links/links-skip.html

```diff
@@ -0,0 +1,39 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Links / Skip Link{% endblock %}
+
+{% block standalone_css %}patterns_skip-link{% endblock %}
+
+{% block content %}
+<a href="#main" class="p-link--skip">Jump to main content</a>
```

Comment:
You should add an element `#main` to this document otherwise Wave conplains.

Reply:
Betrayed by my own devices!